### ISSUE TITLE
Import fails with missing mypyc-compiled file in 4.7 and 4.8 on M1 Macs

### ISSUE THREAD

**Issue by mikeyk** [alisaifee/coredis#116]
Our dev environment uses Linux for prod and M1 Macs locally. As of 4.7.0 and including 4.8.0, Coredis fails to import when running under a pip environment on the M1s, failing with:

```
ModuleNotFoundError: No module named 'b4308b9e21656f7d771e__mypyc'
```

when importing `coredis._packer`.

Installing/importing on Linux works just fine. Anything we can do to help debug?

**Comment by alisaifee** [comment]
I don't immediately see why this would start in 4.7.0 as the version of the [cibuildwheel](https://github.com/pypa/cibuildwheel) github action used to compile the wheels in CI was bumped in 4.6.0. 

To narrow this down a bit:

- Is 4.6.0 working?
- What version of python are you using?
- What is the name of the wheel that gets installed (i.e. which one of these: https://pypi.org/project/coredis/4.7.0/#files)

**Comment by mikeyk** [comment]
* 4.6.0 works great (it's what we've been using recently)
* `Python 3.10.8`
* `coredis-4.6.0-cp310-cp310-macosx_11_0_arm64.whl` is the installed wheel that works
* `coredis-4.8.1-cp310-cp310-macosx_11_0_arm64.whl` doesn't work

Not sure if interesting or expected but I noticed the wheels dropped around ~70kb in size between 4.6 and 4.7

**Comment by alisaifee** [comment]
The main binary isn't present so the drop in size makes sense. 
I just discovered as well that the issue is present on all osx wheels!

**Comment by alisaifee** [comment]
Both 4.7 & 4.8 have patch releases to address this issue in [4.7.1](https://github.com/alisaifee/coredis/releases/tag/4.7.1) and [4.8.2](https://github.com/alisaifee/coredis/releases/tag/4.8.2) respectively.

Though I haven't pinpointed exactly how this started, my guess is that it is due to a build environment change somewhere between the github `macos-latest` runner and way cibuildwheel installs pythons. This fix itself is [here](9750bbc5d26cae618e6ebb15568b7e01952ec92b)

Thanks for reporting this!

**Comment by alisaifee** [comment]
@mikeyk closing this as the wheels look correct now as far as I can tell though I didn’t test on osx + arm specifically
